api/client/network.go

```diff
@@ -237,7 +237,7 @@ func (cli *DockerCli) CmdNetworkInspect(args ...string) error {
 	return nil
 }
 
-// Consolidates the ipam configuration as a group from differnt related configurations
+// Consolidates the ipam configuration as a group from diffrent related configurations
```

Comment:
oh! must've been :zzz: 

that's embarrassing that I didn't spot it

@haoshuwei can you fix that?